src/services/blockchain/__tests__/eth-bc-service.test.ts

```diff
@@ -40,22 +40,19 @@ describe('ETH service',  () => {
   });
 
   test('should connect to local ganache server', async () => {
-    try {
-      await ethBc.connect();
-    } catch (err) {
```

Comment:
@simonovic86 let me know if these try-catch blocks are actually important, but best as I could tell they were just swallowing information about test failures unnecessarily.

Reply:
it's better to remove the `try-catch` 👍